src/Microsoft.Extensions.CodeGeneration.Utils/project.json

```diff
@@ -5,7 +5,8 @@
   "version": "1.0.0-*",
   "dependencies": {
     "Microsoft.CodeAnalysis.CSharp.Workspaces": "1.3.0-*",
-    "Microsoft.DotNet.ProjectModel": "1.0.0-*"
+    "Microsoft.DotNet.ProjectModel": "1.0.0-*",
+    "Microsoft.Extensions.PlatformAbstractions": "1.0.0-*"
```

Comment:
Isn't `PlatformAbstractions` brought in by `"Microsoft.DotNet.ProjectModel": "1.0.0-*"`